CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+## [0.3.6] - 2022-05-30
+### Changed
+- Dependency changes: Added support to ActiveSupport 6.1, changed faraday version to v1.0.
```

Comment:
Please also link the PR as per the other entries.  The reason this is nice (specifically if you copy this into the commit message as per my other comment) is that github will show in each PR which release it went out it by nature of a commit referencing a PR.

Example from a different repo:
![Switch_to_GitHub_Actions_by_Fryguy_·_Pull_Request__105_·_ManageIQ_more_core_extensions](https://user-images.githubusercontent.com/52120/165988657-04ad4f6f-d0ea-4bc2-a343-1bafeb067c49.png)